src/OpenLoco/src/World/Company.cpp

```diff
@@ -335,6 +335,139 @@ namespace OpenLoco
         company->observationTimeout = 5;
     }
 
+    /* 0x004A6841
+     * Creates a vector of all the available rail track (trains and trams) for a company
+     * Tram track is marked with a (1<<7) flag within the uint8_t
+     */
+    sfl::static_vector<uint8_t, Limits::kMaxRoadObjects + Limits::kMaxTrackObjects> companyGetAvailableRailTracks(const CompanyId id)
```

Comment:
I think we should add a type alias for this tbh. It's used several times in Company.h/cpp and once in ToolbarTop.cpp, and very verbose.

Reply:
Okay done.